code401challenges/src/main/java/code401challenges/Library.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package code401challenges;
+
+public class Library {
```

Comment:
Not a big deal, you're not losing points for it, but since you're not using this Library file, it's good practice to remove it from your repo.